tony-core/src/test/java/com/linkedin/tony/TestTonyConfigurationFields.java

```diff
@@ -29,6 +29,7 @@ public void initializeMemberVariables() {
     xmlPropsToSkipCompare.add(TonyConfigurationKeys.getMemoryKey(Constants.WORKER_JOB_NAME));
     xmlPropsToSkipCompare.add(TonyConfigurationKeys.getVCoresKey(Constants.WORKER_JOB_NAME));
     xmlPropsToSkipCompare.add(TonyConfigurationKeys.getGPUsKey(Constants.WORKER_JOB_NAME));
+    configurationPropsToSkipCompare.add(TonyConfigurationKeys.DOCKER_IMAGE);
```

Comment:
Why do you use `configurationPropsToSkipCompare` here instead of `xmlPropsToSkipCompare`? What's the difference?

Reply:
I think configurationPropsToSkip ignores the confs in TonyApplicationKeys, xmlPropsToSkip ignores confs in tony-default.xml

Reply:
The comment above is right